### PR TITLE
Shapovalov Viacheslav - laba 4 - 12.24

### PR DESCRIPTION
**Pull request by shap-po**

### CODE REVIEW

File: 4.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+def _12_24():
```
Comment (mecchmatProjects) [inline 4.py:1]: please, submit also test file

File: 4.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    counter = {}
+    for cube in cubes:
+        if cube[0] == 3 and cube[2] == 'wood':
+            counter['wood'] = counter.get('wood', 0) + 1
+        if cube[0] == 5 and cube[2] == 'metal':
+            counter['metal'] = counter.get('metal', 0) + 1
+    print(counter)
+
+
+_12_24()
```
Comment (mecchmatProjects) [inline 4.py:55]: Please add, if __name__ == "__main__":

Comment (shap-po) [reply 4.py:55]: no

**Review by mecchmatProjects — APPROVED** [review]
+1

**Pull request merged**